src/posit/connect/client.py

```diff
@@ -6,6 +6,7 @@
 
 from requests import Response, Session
 
+from posit.connect.oauth.oauth import OAuth
```

Comment:
Could we avoid the double `.oauth.oauth` by moving the contents of `oauth/oauth.py` into `oauth/__init__.py` ? Or we could redefine how these classes are exported by importing in __init__.py

This answer describes an alternative: https://stackoverflow.com/a/35733111

Reply:
Nvm you beat me to it!